test/unit/specs/highlightChunks.spec.js

```diff
@@ -158,4 +158,23 @@ describe('highlightChunks', () => {
       },
     ]);
   });
+  test('should match whole word not part of other word', () => {
+    const text = 'example amp';
+    const string = 'amp';
+
+    const chunks = highlightChunks(text, string, { wholeWordMatch: true });
+    console.log(chunks);
```

Comment:
Can help to remove this `console.log` and add a test case for digits?

Reply:
done